service/app/validators/email_validator.rb

```diff
@@ -0,0 +1,7 @@
+class EmailValidator < ActiveModel::EachValidator
+	def validate_each(record, attribute, value)
+		unless value =~ /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+			record.errors[attribute] << (options[:message] || "is not email addreess")
```

Comment:
__[rubocop]__
* Tab detected. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)
* Use 2 (not 1) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)